### PR TITLE
feat: german translations for dynamax

### PR DESCRIPTION
**Pull request by ReuschelCGN**

### CODE REVIEW

File: packages/locales/lib/human/de.json [hunk 1]
```diff
@@ -763,5 +763,22 @@
...
+  "started": "gestartet",
+  "ended": "beendet",
+  "global_search_stations": "Füge den Namen der Kraftquelle ein...",
+  "station_timers": "Kraftquellen Timer",
+  "station_opacity": "Kraftquellen Durchsichtigkeit",
+  "max_battles": "Dyna-Raids",
+  "dynamax": "Dynamax",
+  "stations_subtitle": "Kraftquellen auf der Map anzeigen",
+  "dynamax_subtitle": "Dyna-Raids auf der Map anzeigen",
+  "override": "Dyna-Raid Override"
```
Comment (Fabio1988) [inline packages/locales/lib/human/de.json:783]: is this "override" just used for Dyna-Raid or for every other preset-filter like raids?

File: packages/locales/lib/human/de.json [hunk 2]
```diff
@@ -763,5 +763,23 @@
...
   "best_buddy": "Bester Kumpel",
-  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}"
+  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}",
+  "unknown_station": "unbekannte Kraftquelle",
+  "exclude_battle": "Dyna-Raids ausschließen",
+  "station": "Kraftquelle",
+  "stations": "Dynamax",
+  "stations_options": "Kraftquellen Optionen",
+  "all_stations": "alle Kraftquellen",
+  "search_battles": "Suche Dyna-Raids",
```
Comment (Fabio1988) [inline packages/locales/lib/human/de.json:773]: Why not being consistent with search phrases? :)

Comment (TurtIeSocks) [reply packages/locales/lib/human/de.json:773]: It's the same in English. Stations/Battles used in backend, translated to Power Spots/Dynamax or Max Battles in clients.

Comment (Fabio1988) [reply packages/locales/lib/human/de.json:773]: @TurtIeSocks was more about the translation itself, he changed the other search_key values, but this matches the old phrasing :) 
it's not about dyna/spots or something else :)

File: packages/locales/lib/human/de.json (hunk 1 above)
Comment (Fabio1988) [reply packages/locales/lib/human/de.json:784]: @TurtIeSocks did you publish this key on purpose like that? instead `dyna_override` like for `raid_override` ? :)

File: packages/locales/lib/human/de.json (hunk 2 above)
Comment (TurtIeSocks) [reply packages/locales/lib/human/de.json:773]: I left it generic in this case. In English it's just `Override` without any category attached to it. Dynamax or Max Battles as a prefix was just too long and feels unnecessary with the way the UI is setup.

File: packages/locales/lib/human/de.json [hunk 3]
```diff
@@ -763,5 +763,25 @@
   "enter_translation": "Übersetzung eingeben",
   "individual_filters": "individuell gefiltert",
   "best_buddy": "Bester Kumpel",
-  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}"
+  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}",
+  "unknown_station": "unbekannte Kraftquelle",
+  "exclude_battle": "Dyna-Raids ausschließen",
+  "station": "Kraftquelle",
+  "stations": "Dynamax",
```
Comment (Fabio1988) [inline packages/locales/lib/human/de.json:770]: Singular = Kraftquelle
plural = Dynamax ?
I think this should be "Kraftquellen"

**Comment by ReuschelCGN** [comment]
> Can you remove these changes from this PR? As they're unrelated to the core goal and I plan to implement this for stations a bit differently.

https://github.com/WatWowMap/ReactMap/pull/1054/files/6c9a3c964c7bc972534679c80c4a9e6a30884d5a#diff-1ecdf5ede1d9677965b661f4a7c5ca4788fb94e8f67bb70d47f2e2910d7ad09e

this?

**Comment by TurtIeSocks** [comment]
> https://github.com/WatWowMap/ReactMap/pull/1054/files/6c9a3c964c7bc972534679c80c4a9e6a30884d5a#diff-1ecdf5ede1d9677965b661f4a7c5ca4788fb94e8f67bb70d47f2e2910d7ad09e
> 
> this?

Right

**Review by TurtIeSocks — APPROVED** [review]

File: packages/locales/lib/human/de.json [hunk 4]
```diff
@@ -763,5 +763,26 @@
...
-  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}"
+  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}",
+  "unknown_station": "unbekannte Kraftquelle",
+  "exclude_battle": "Dyna-Raids ausschließen",
+  "station": "Kraftquelle",
+  "stations": "Kraftquellen",
+  "stations_filters": "Dynamax Filter Einstellungen",
+  "stations_options": "Kraftquellen Optionen",
+  "all_stations": "alle Kraftquellen",
+  "search_battles": "Suche Dyna-Raids",
```
Comment (Fabio1988) [inline packages/locales/lib/human/de.json:774]:
```suggestion
  "search_battles": "Dyna-Raids durchsuchen",
```

File: packages/locales/lib/human/de.json [hunk 5]
```diff
@@ -533,11 +533,11 @@
...
   "refresh": "Aktualisieren",
-  "search_raids": "Suche nach Raids",
-  "search_eggs": "Suche nach Eier",
-  "search_quests": "Suche nach Quests",
-  "search_lures": "Suche nach Lockmodulen",
-  "search_invasions": "Suche nach Invasionen",
+  "search_raids": "Raids durchsuchen",
+  "search_eggs": "Eier durchsuchen",
+  "search_quests": "Quests durchsuchen",
+  "search_lures": "Lockmodulen durchsuchen",
```
Comment (Fabio1988) [inline packages/locales/lib/human/de.json:539]:
```suggestion
  "search_lures": "Lockmodule durchsuchen",
```

**Comment by Fabio1988** [comment]
One more comment.. but I think you need to rebase on latest and maybe drop zhe commits of the tsx file which have been reverted first... Guess those are causing the conflict

**Review by Fabio1988 — APPROVED** [review]

**Pull request merged**